selectors.go

```diff
@@ -62,11 +68,22 @@ func ChainIdFromSelector(chainSelectorId uint64) (uint64, error) {
 
 func SelectorFromChainId(chainId uint64) (uint64, error) {
 	if chainSelectorId, exist := evmChainIdToChainSelector[chainId]; exist {
-		return chainSelectorId, nil
+		return chainSelectorId.ChainSelector, nil
 	}
 	return 0, fmt.Errorf("chain selector not found for chain %d", chainId)
 }
 
+func NameFromChainId(chainId uint64) (string, error) {
```

Comment:
For completeness, may we have a `func ChainIdFromName`?
It'd be useful for Atlas ccip apps.